src/views/UserProfile.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Header from "../Components/Header";
+import { useContext } from "react";
+import { AuthContext } from "../store/AuthContext";
+
+function UserProfile(){
+    const {user} = useContext(AuthContext)
+
+    return (
+      <div>
+          <Header/>
+          <h2>Welcome {user?.userName} to my App</h2>
```

Comment:
the user object has no name (yet?)
```suggestion
          <h2>Account {user?.email}</h2>
```